detections/endpoint/windows_cisco_secure_endpoint_stop_immunet_service_via_sfc.yml

```diff
@@ -0,0 +1,58 @@
+name: Windows Cisco Secure Endpoint Stop Immunet Service Via Sfc
+id: 44badcb1-2e8c-4628-9537-021bbae571ad
+version: 1
+date: '2025-01-08'
+author: Nasreddine Bencherchali, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects the use of the `sfc.exe` utility, in order to stop the Immunet Protect service. The Sfc.exe utility is part of Cisco Secure Endpoint installation. This detection leverages telemetry from the endpoint, focusing on command-line executions involving the `-k` parameter. This activity is significant as it indicates potential tampering with defensive mechanisms. If confirmed malicious, attackers could partially blind the EDR, enabling further compromise and lateral movement within the network.
+data_source:
+- Sysmon EventID 1
+- Windows Event Log Security 4688
+- CrowdStrike ProcessRollup2
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes where Processes.process_name="sfc.exe" Processes.process="* -k*" by Processes.dest Processes.user Processes.parent_process Processes.process_name Processes.parent_process_name Processes.process Processes.process_id Processes.parent_process_id | `drop_dm_object_name(Processes)` | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)` | `windows_cisco_secure_endpoint_stop_immunet_service_via_sfc_filter`'
```

Comment:
Do we have any other fields (process_path, perhaps?) that we can add an extra conditional onto in order to avoid collisions with the System File Checker (also `sfc.exe`)? We've had issues in the past with detections on the System File Checker throwing massive false positives on the Cisco executable. I don't _think_ `-k` is a flag that works for the system file checker, but the wildcards in `* -k*` make me nervous that it won't matter.

Reply:
I check with the actual sfc.exe flag and as you guessed it does not have the `-k` the reason i didn't add a path was because you could use this binary from anywhere technically via a copy paste.

I'll simply exclude the built-in sfc path and we should good to go.